scalyr_agent/configuration.py

```diff
@@ -701,6 +701,7 @@ def print_useful_settings(self, other_config=None):
             "default_sessions_per_worker",
             "default_worker_session_status_message_interval",
             "enable_worker_session_process_metrics_gather",
+            "ignore_checkpoints_on_startup_path_globs",
```

Comment:
I'll remove this for now under Python 2.7 since there is a bug with JsonArray under Python 2.7 which is pain to fix and not worth doing it since we will drop support for 2.7 in the future anyway.